test/parallel/test-assert-deep.js

```diff
@@ -375,7 +375,11 @@ assertOnlyDeepEqual(
   new Map([[undefined, null], ['+000', 2n]]),
   new Map([[null, undefined], [false, '2']]),
 );
-
+const xarray = ['x'];
+assertOnlyDeepEqual(
```

Comment:
Shouldn’t the sets be strict deep equal and regular deep equal? Neither `deepEqual` nor `deepStrictEqual` should throw in this case.
```suggestion
assertDeepAndStrictEqual(
```